internal/gapicgen/generator/gapics.go

```diff
@@ -30,6 +32,13 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+var (
```

Comment:
There appear to be generator changes, not just a bulk generation. Is that intentional?

Reply:
Yes, I tried to divide this up in some of the early commits to show what I was doing. I added in the ability to generate a new module when first generating a client. Sorry if this was not clear in the PR message I will update.